### ISSUE TITLE
Top level spaces have cursor: grab styling

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#18951]
### Steps to reproduce

Hover over a top level space in the space panel

### What did you expect?

The cursor should be `pointer`:

![pointer](https://user-images.githubusercontent.com/48614497/132546678-20891a44-6d01-4f16-afb2-4613bf7ac955.gif)

### What happened?

The cursor appeared as `grab`:

![grab](https://user-images.githubusercontent.com/48614497/132546748-baee12c8-1c44-40c3-b885-eeea5c10cbba.gif)

### Operating system

NixOS unstable

### Browser information

Firefox 91.0.1

### URL for webapp

develop.element.io

### Homeserver

townsendandsmith.ml

### Have you submitted a rageshake?

No
